engines/jsonengine.py

```diff
@@ -77,9 +78,6 @@ def disconnect(self):
                 current_output_file.write('\n]')
                 current_output_file.close()
             except:
```

Comment:
Given the comment here we can probably just remove the try/except piece of this and just keep the code in the try. I'm OK with not doing that now, but if that's what you want to do then let's not remove the comment since that will make figuring out why the error catching code is there.